circle.yml

```diff
@@ -59,7 +59,9 @@ stages:
 
       - type: shell
         name: NPM lint
-        command: npm run lint
+        # Note that we run the relay build first in order to expose autogenerated
+        # GraphQL types to `flow`.
+        command: npm run build:relay && npm run lint
```

Comment:
why do flow types matter for the linter of our eslint config is free of any flow-specific stuff?

Reply:
Types matter for `flow` which is currently bundled into the `lint` command. We could make this more clear by separating `flow` out from `lint`, since they're not really all that related semantically.